Tasks/Vadzim/Task3/query04.sql

```diff
@@ -0,0 +1,15 @@
+﻿--Query 4
+SELECT
+    Name
+FROM Person.StateProvince as p
+WHERE 
+    StateProvinceID IN (SELECT StateProvinceID FROM Person.Address 
+        WHERE AddressID IN (SELECT AddressID FROM Person.BusinessEntityAddress AS A 
+            WHERE BusinessEntityID IN (SELECT BusinessEntityID FROM Purchasing.Vendor)))
+    AND 
+    NAME NOT IN (SELECT Name FROM Person.StateProvince
```

Comment:
usage of Name column for filtering creates additional subquery -> that would add extra unneccessary complexity to query
also even if tasks is about subqueries it would be nice to avoid where they could be avoided